integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -610,6 +625,9 @@ private int statfsInternal(String path, Statvfs stbuf) {
     if (res != 0) {
       return res;
     }
+    if (mConf.getBoolean(PropertyKey.USER_UFS_ENABLED)) {
```

Comment:
if this method is called frequently, we could cache the value mConf.getBoolean(PropertyKey.USER_UFS_ENABLED) because the call of mConf.getXXX might be expensive for lock acquiring

Reply:
Will add a boolean and store the value. Reconsidering using options to save Fuse values